### PR TITLE
Add Breezometer API 

### PR DESCRIPTION
**Pull request by jayeshchoudhary**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by jayeshchoudhary** [comment]
@pawelborkar can you please review  
this PR is under hactoberfest

File: README.md [hunk 1]
```diff
@@ -476,6 +476,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [AirVisual](https://airvisual.com/api) | Air quality and weather data | `apiKey` | Yes | Unknown |
+| [Breezometer](https://docs.breezometer.com/api-documentation/pollen-api/v2/#daily-forecast) | Daily Forecast pollen conditions data for a specific location | `apiKey` | Yes | Unknown |
```
Comment (matheusfelipeog) [inline README.md:479]: - The name of this API is BreezoMeter Pollen
- Remove anchor from link

```suggestion
| [BreezoMeter Pollen](https://docs.breezometer.com/api-documentation/pollen-api/v2/) | Daily Forecast pollen conditions data for a specific location | `apiKey` | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @jayeshchoudhary 😄 

Can you resolve this review, please?

**Comment by jayeshchoudhary** [comment]
done @matheusfelipeog

**Pull request merged**